src/data/attribute.rs

```diff
@@ -128,7 +129,7 @@ pub fn set_attribute(attr: &str, value: &[u8]) {
 pub fn store_metadata(metastruct: &Struct) {
     let metadata = process_metadata(metastruct, String::new());
     for (key, value) in metadata {
-        let attr = format!("{KUADRANT_NAMESPACE}\\.{key}");
+        let attr = format!("{KUADRANT_NAMESPACE}\\.auth\\.{key}");
         debug!("set_attribute: {attr} = {value}");
```

Comment:
What about this `debug!` macro in `pub fn set_attribute(attr: &str, value: &[u8])` ?